tests/api/search/entities.test.js

```diff
@@ -110,6 +110,13 @@ describe('search:entities', () => {
         _.map(results, 'uri').should.containEql(work.uri)
       })
     })
+
+    describe('not exact', () => {
+      it('should match flattened terms', async () => {
+        const results = await search({ types: 'series', search: 'ဟယ်ရီပေါ်တာ' })
```

Comment:
same as last comment what about finding the first key which is not in `activeI18nLangs` ?

Reply:
done in 1b4ec358